### DISCUSSION TITLE
Zigbee TRV recommandation

### DISCUSSION THREAD

**Discussion by jiquem** [jmcollin78/versatile_thermostat#502] [Q&A] [answered]
Hello
I currently have 5 Aqara SRTS-A01.
They are not 100% compatible with Versatile and batteries need to be changed every two months. 
Do you have recommandation to change them ?

**Comment by hnb2907** [comment, marked as answer]
Hi, I'm using 6 of Sonoff TRVZB.  They were in use while the heating was on last winter in the UK for about 4-5 months.  For the past 6 months they've been idling.  They are all reporting >96% battery remaining today (I'm not sure if this is accurate).  At the moment, they are still running on the batteries that I originally installed.

Bought them from Sonoff store, while they had a promotion, maybe black friday, I can't remember!?


Hope that helps?
Chris.

**Reply by zdeseb** [reply]
Hi, is it possible to use Sonoff TRVZB directly with Home Assistant (running on Raspberry Pi) or do I need yet another hardware (some kind of HUB) to make it work?

Thanks,
Zdenek

**Reply by hnb2907** [reply]
I've got my TRVZB's working on homeassistant with this https://sonoff.tech/product/gateway-and-sensors/sonoff-zigbee-3-0-usb-dongle-plus-e/

Hope that helps.
C.

**Reply by zdeseb** [reply]
Thanks. I am just wondering whether it is the right solution for me because Home Assistant Raspberry Pi is placed behind a few walls so there could be problem with signal between the valve and the USB dongle.

Could you compare the wireless Zigbee 3.0 and the WiFi connection range for me to be able to imagine which placement could be acceptable?

**Reply by zdeseb** [reply]
Maybe I'll try to find a valve which could work directly on WiFi without a hub because WiFi signal is already everywhere I need.

**Reply by hnb2907** [reply]
I think zigbee and wifi are both 2.4ghz, and have similar RSSI's around my house.  

We live in a 3 bedroom 1950's brick built house in the UK.  The TRV's are on the ground floor and first floor.  The zigbee hub is in the loft (floor 2).  The signal travels through multiple ceilings and brick walls to the furthest locations.
The TRV's are reporting between -52dbm to -77dbm.  I'm not seeing any problems with zigbee communications, or interference with wifi.

If your wifi signal is good across all areas of your property, I would think zigbee will be similar.


C.

**Reply by zdeseb** [reply]
That's the problem. WiFi had not been good enough and I had to create a WiFi mesh with multiple units to solve it. That's why I am quite cautious regarding Zigbee.

Anyway. Thank you for sharing your experience.

Z.

**Reply by dakota** [reply]
Zigbee is a meshed network (a true mesh, rather than the pseudo-mesh that WiFi uses).

You simply need to add mains powered zigbee devices (like smart plugs or smart bulbs), they act as routers and help extend the network.

**Comment by johnbur1** [comment]
+1 for Sonoff. Used 15 of them all last winter and they have been flawless. Added advantage is that they work in over_valve mode with VT

**Comment by jmcollin78** [comment]
You also have Shelly TRV that are fully compatible with the 'over_valve' mode of VTherm which is the best mode for regulation.

If your battery are draining rapidly you can adjust some parameters to avoid changing to frequently the valve openness. Look at the readme.
